x-pack/plugins/apm/public/components/shared/MetadataTable/index.tsx

```diff
@@ -77,14 +80,21 @@ export function MetadataTable({ sections }: Props) {
           />
         </EuiFlexItem>
       </EuiFlexGroup>
+      {isLoading && (
+        <EuiFlexGroup justifyContent="center">
+          <EuiFlexItem grow={false}>
+            <EuiLoadingSpinner />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      )}
       <HeightRetainer>
         {filteredSections.map((section) => (
           <div key={section.key}>
             <EuiTitle size="xs">
               <h6>{section.label}</h6>
             </EuiTitle>
```

Comment:
I thought we'd force lowercase capitalization for the section titles so we'd avoid the odd-cases like `Http` > `http` which is true to the raw formatting in the fieldnames. I think it'll make it easier to scan.
![CleanShot 2021-09-30 at 09 15 08@2x](https://user-images.githubusercontent.com/4104278/135405731-96a36939-a67d-45f7-b1e0-0e415d11565f.png)

Reply:
yes, on my list!